src/components/create-card.jsx

```diff
@@ -1,9 +1,76 @@
 import React from 'react';
 
 class CreateCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      question: '',
+      answer: ''
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleReset = this.handleReset.bind(this);
+  }
+
+  handleReset() {
+    this.setState({
+      question: '',
+      answer: ''
+    });
+    this.props.setView('view-cards');
+  }
+
+  handleChange(event) {
+    if (event.target.name === 'question') {
+      this.setState({
+        question: event.target.value
+      });
+    } else {
+      this.setState({
+        answer: event.target.value
+      });
+    }
+  }
```

Comment:
This is okay, but not very flexible if you need to add more inputs. Notice the name property value matches the property in state you are updating so you can just use that information to consolidate this method and make it more versatile at the same time.

FOR EXAMPLE
```JAVASCRIPT
handleChange(event) {
  const { name, value } = event.target;

  this.setState({
    [name]: value
  });
}
```